src/Pim/Bundle/DataGridBundle/Datasource/ProductDatasource.php

```diff
@@ -101,7 +108,7 @@ protected function initializeQueryBuilder($method, array $config = [])
         $factoryConfig['default_locale'] = $this->getConfiguration('locale_code');
         $factoryConfig['default_scope'] = $this->getConfiguration('scope_code');
         $factoryConfig['limit'] = $this->getConfiguration(ContextConfigurator::PRODUCTS_PER_PAGE);
-        $factoryConfig['search_after'] = null; // TODO with TIP-664
+        $factoryConfig['search_after'] =  $this->getConfiguration('search_after', false);
```

Comment:
the second param is not to defined a default value, it's to define if value is require or not